README.md

```diff
@@ -1,3 +1,17 @@
+## Install
+```
+git clone https://github.com/ljonka/mediawiki-extensions-Math.git extensions/Math
+echo "wfLoadExtension(\"Math\"); \$wgDefaultUserOptions[\"math\"] = \"mathjax\";" >> LocalSettings.php
+php maintenance/update.php --quick
+```
+
+## Enable MathJax Service
+```
+npm install -g pm2
+cd extensions/Math && npm update && pm2 start server && cd ../../
```

Comment:
no, this is a recomendation to start a nodejs server and run the service as a background job.